pymoskito/simulation_gui.py

```diff
@@ -1503,16 +1499,25 @@ def find_all_plot_docks(self):
 
         return list
 
-    def export_csv(self, plot_item, name, *args):
-        exporter = pg.exporters.CSVExporter(plot_item)
-        filename = QFileDialog.getSaveFileName(self,
-                                               "CSV export", name + ".csv",
-                                               "CSV Data (*.csv)")
-        if not filename[0]:
-            return
+    def exportPlotItem(self, plot_item, name, coord_item, time_item):
+        dataPoints = {}
+        for i, c in enumerate(plot_item.curves):
+            if c.getData() is None:
+                continue
+            if len(c.getData()) > 2:
+                self._logger.warning('Can not handle the amount of data!')
+                continue
+            dataPoints[c.name() + str('.0')] = c.getData()[0]
```

Comment:
While not completely impossible, I would be really surprised if the x-data should differ between different curves of the same plots. Maybe the x-data of the first one could be exported as a separate column `time` (if not already present)?